src/3d/qgs3daxis.cpp

```diff
@@ -918,7 +898,7 @@ Qt3DExtras::QText2DEntity *Qgs3DAxis::addCubeText( const QString &text, float te
   return textEntity;
 }
 
-void Qgs3DAxis::createAxis( Qt::Axis axisType )
+void Qgs3DAxis::createAxis( const Qt::Axis &axisType )
```

Comment:
```suggestion
void Qgs3DAxis::createAxis( Qt::Axis axisType )
```

It's actually preferable to pass small and trivially copiable types like enums by value, not reference.

Reply:
Dropped.